packages/interactivity-router/src/index.ts

```diff
@@ -209,16 +209,18 @@ const isValidEvent = ( event: MouseEvent ) =>
 // Variable to store the current navigation.
 let navigatingTo = '';
 
+const navigationTexts = {
+	loadedFromServer: false,
+	loading: 'Loading page, please wait.',
+	loaded: 'Page Loaded.',
```

Comment:
Why did we move the `loading` & `loaded` out of the store? Do we purposefully not want them to be reactive?

Reply:
These were always static texts, there was no reason for them to be in the store, it was just a mechanism to pass server data to the client.

Reply:
Right, that makes sense, thanks! 🙂